app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<ul class="userCards py-8 w-[80%] mx-auto flex flex-col gap-8">
+  <% @users.each do |user| %>
+    <li class="userCard">
+      <a href="users/<%= user.id %>" class="flex flex-row gap-[30px]">
+        <div class="userImage w-[15%]">
+          <img src=<%= user.photo %>>
+        </div>
+        <div class="userDetails bg-[#11537d] rounded-[30px] flex-grow text-white flex justify-between items-center p-[30px]">
+          <h2 class="text-[40px] font-[600]"><%= user.name %></h2>
+          <p class="self-end text-[20px] font-[400]">Number of posts: <%= user.posts_counter %></p>
+        </div>
+      </a>
+    </li>
+  <% end %>
```

Comment:
[OPTIONAL]

- You can use `app/views/users/_user.html.erb` for better practice instead of using `@objects.each`.

```ruby
# app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

What it will do, it will render each user one by one and display all the user that comes from the controller.